docs/en/queries/transactions.md

```diff
@@ -38,12 +38,14 @@ try {
         [3, 'db'],
     ];
         
-    foreach ($tags as [$id, $name] => $tag) {
+    foreach ($tags as list($id, $name)) {
```

Comment:
Why? It's the same, isn't it?

Reply:
On my local env with win10, apache, PHP 8.1.1 it throws error. list() does not.